examples/workspaces/main.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_subnet" "private-a" {
+  vpc_id            = "${aws_vpc.main.id}"
+  availability_zone = "us-east-1b"
```

Comment:
Both of these subnets are in `us-east-1b`

Reply:
Fixed